lib/code_corps/github/adapters/github_repo.ex

```diff
@@ -0,0 +1,20 @@
+defmodule CodeCorps.GitHub.Adapters.GithubRepo do
```

Comment:
Thinking about moving this to `lib/code_corps/github_repo/adapter.ex`. According to 

![image](https://user-images.githubusercontent.com/1925540/27636214-ac38af76-5c0a-11e7-9866-eab9655b08a6.png)

That's the place where it should be going, assuming `web/models/github_repo.ex` will be moved into `lib/code_corps/github_repo.ex` once we upgrade to 1.3.

That would provide us the namespace of `CodeCorps.GithubRepo.Adapter`. In the place where we use it right now, we could just alias `CodeCorps.GithubRepo` and then call `GithubRepo.Adapter.from_api`, which seams a bit nicer than what we have.